index.html

```diff
@@ -65,13 +65,13 @@ <h1>Bikes for Refugees</h1>
               seekers in Scotland.
             </p>
             <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="button_vol"> Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
+            <div class="learn_more">
               <div class="article">
```

Comment:
You can use an `article` tag for to be more semantic.

Also, there is a lack of spacing between the content of the div and the border (see the screenshot of the homework). Do you know which CSS property that handles this? Please add it :)